CONTRIBUTING.md

```diff
@@ -36,8 +36,16 @@ Thank you! We'll try to respond as quickly as possible.
 
 ## Submitting feature requests
 
-If you can't find an issue (open or closed) describing your idea on our [issue
-tracker], open an issue. Adding answers to the following
+Diesel's issue tracker is meant to represent our current roadmap.
+An open issue represents either a bug, or a new feature that a member of the
+Diesel team is actively working on.
+
+This means that you should not submit a feature request to our issue tracker,
+unless you were asked to do so by a member of the Diesel team.
+Feature requests should instead be posted at discourse.diesel.rs.
+
+If you can't find thread describing your idea on our [discourse],
```

Comment:
"find [a] thread"

Reply:
Maybe call discourse "forum" here because not everyone will know that this is the name of a forum software. This reads super weird if you read the "discourse" in "describing your idea on our discourse" as "conversation" or "discussion" instead of "forum".